core/drivers/apic.c

```diff
@@ -131,13 +131,30 @@ static bool send_self_ipi(uint32_t vector)
 
 static void lapic_eoi(void)
 {
-	/* TODO: will figure out how to support X2 APIC*/
-	lapic_x1_write_reg(LAPIC_EOI, 1);
+	uint64_t apic_base_msr = read_msr(MSR_APIC_BASE);
+	if (!(apic_base_msr & LAPIC_ENABLED)) {
```

Comment:
braces {} are not necessary for single statement blocks.

Reply:
see line:118~120 https://github.com/op-tee-ia/optee_os/blob/9d86f5d4f61e068bc22f9cbe55741a001eadf3b1/core/drivers/apic.c#L118

I want to keep the style same as originals. So you sure no braces for single line blocks? if so, I can modify it.

Reply:
We are following linux kernel coding styles. These are warnings detected by script: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/scripts/checkpatch.pl

Reply:
@YadongQi  The newly added code needs to follow the style of the Linux kernel,
and the old code problem will soon be fixed

Reply:
Fine, I will update the patch.